### PR TITLE
Updated buildscript for samsam package

### PR DESCRIPTION
**Pull request by saurabhghumnar**
Updating samsam package buildscript for version typo

Part of [#1220](https://github.com/ppc64le/build-scripts/pull/1220)

### CODE REVIEW

**Comment by saurabhghumnar** [comment]
@Siddhesh-Ghadi Please review

File: s/sinonjs__samsam/sinonjs__samsam_rhel_8.3.sh [hunk 1]
```diff
@@ -16,7 +18,7 @@
 # ----------------------------------------------------------------------------
 
 PACKAGE_NAME=@sinonjs/samsam
-PACKAGE_VERSION=5.3.0
+PACKAGE_VERSION=v5.3.0
 PACKAGE_URL=https://github.com/sinonjs/samsam
 
 yum -y update && yum install -y yum-utils nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git gcc gcc-c++ libffi libffi-devel ncurses git jq make cmake
```
Comment (Siddhesh-Ghadi) [inline s/sinonjs__samsam/sinonjs__samsam_rhel_8.3.sh:24]: Are all these packages required?

File: s/sinonjs__samsam/sinonjs__samsam_rhel_8.3.sh [hunk 2]
```diff
@@ -1,9 +1,11 @@
 # -----------------------------------------------------------------------------
 #
 # Package	: @sinonjs/samsam
-# Version	: 5.3.0
+# Version	: v5.3.0
 # Source repo	: https://github.com/sinonjs/samsam
 # Tested on	: RHEL 8.3
```
Comment (Siddhesh-Ghadi) [inline s/sinonjs__samsam/sinonjs__samsam_rhel_8.3.sh:6]: Can we add ubi?
```suggestion
# Tested on	: RHEL 8.3, UBI 8.5
```

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]

File: s/sinonjs__samsam/sinonjs__samsam_rhel_8.3.sh (hunk 1 above)
Comment (saurabhghumnar) [reply s/sinonjs__samsam/sinonjs__samsam_rhel_8.3.sh]: Removed unnecessary package installations

**Pull request closed without merging**